tests/test_qnode_legacy.py

```diff
@@ -1096,19 +1117,22 @@ def circuit(a, shots=0):
             circuit = QNode(circuit, dev)
 
         assert len(circuit(0.8)) == 10
-        assert circuit.qtape.operations[0].wires.labels == (0,)
+        tape = qml.workflow.construct_tape(circuit)(0.8)
+        assert tape.operations[0].wires.labels == (0,)
 
         assert len(circuit(0.8, shots=1)) == 10
-        assert circuit.qtape.operations[0].wires.labels == (1,)
+        tape = qml.workflow.construct_tape(circuit)(0.8, shots=1)
+        assert tape.operations[0].wires.labels == (1,)
 
         assert len(circuit(0.8, shots=0)) == 10
-        assert circuit.qtape.operations[0].wires.labels == (0,)
+        tape = qml.workflow.construct_tape(circuit)(0.8, shots=0)
+        assert tape.operations[0].wires.labels == (0,)
 
     # pylint: disable=unexpected-keyword-arg
     def test_no_shots_per_call_if_user_has_shots_qfunc_arg(self):
         """Tests that the per-call shots overwriting is suspended
         if user has a shots argument, but a warning is raised."""
-        dev = qml.device("default.mixed", wires=[0, 1], shots=10)
+        dev = qml.device("default.qubit", wires=[0, 1], shots=10)
```

Comment:
This file tests legacy devices, hence why `default.mixed` is used. Now that `default.mixed` is being ported over, we might need a different device. Maybe we should use the `tests/default_qubit_legacy` device? cc @albi3ro

Reply:
After speaking with @albi3ro, I think it will be addressed in a follow-up PR. 😄